### ISSUE TITLE
Problem when setup the database with datasets

### ISSUE THREAD

**Issue by CeVague** [codalab/chalab#48]
"make dataset" work like always (default datasets appeared in the list) but when you build your challenges, it throw the error witch say that files aren't found.

**Comment by madclam** [comment]
this is related to issue 47

**Comment by madclam** [comment]
![image](https://user-images.githubusercontent.com/7594063/29881738-45c37bae-8d60-11e7-9ba1-d14eab08bbd7.png)

**Comment by Tthomas63** [comment]
If the datasets are not loaded in and stored in their raw-uncompressed format in the media directory, building competitions which reference them will fail.

Currently have Azure working so we can upload the default datasets on one instance, and have them available to load in with fixtures. ( Referencing cloud files instead of local files ).

This should make the default datasets much easier to load in without having to directly copy them to the necessary directory before hand.

*Note: This would depreciate `load_chalearn_dataset`, and instead we would simply use a Django Fixture ( included by default ) which references these files on the azure server. It would only be necessary to run `load_chalearn_dataset` if the datasets themselves were updated and we needed to update the remote files.*
